### ISSUE TITLE
PPO and SAC models for Navier-Stokes

### ISSUE THREAD

**Issue by HanjiangHu** [lukebhan/PDEControlGym#24]
Thanks for the amazing work! I have tried to train the PPO and SAC models for Navier-Stokes PDE but the reward is still around -40 and cannot get up to -5.3 and -17.8 as reported in Table 3 in the paper. I didn't change any training code and gym environment for Navier-Stokes and it seems that the action is automatically normalized to [-1,1] from the released code. Could you please share the trained RL models for Navier-Stokes with the performance in the paper? That will be very helpful. Thanks!

**Comment by lukebhan** [comment]
Hi, 
Sorry for the late response and thanks for reaching out. The PPO and SAC models for the Navier Stokes PDE require the initial conditions to be set to 0 aka modify the function 
```
# Set initial condition function to be zero
def getInitialCondition(X):
    u = np.zeros_like(X) 
    v = np.zeros_like(X) 
    p = np.zeros_like(X) 
    return u, v, p
```

My collaborator @alwaysbyx  has also made a Jupyter notebook in the examples folder on the dev branch for the NS example to help clarify. The models are uploaded to https://huggingface.co/lukebhan/PDEControlGymModels. Please let me know if you have any issues/questions.

**Comment by HanjiangHu** [comment]
Thank you very much! That helps a lot.
